---
string[] picture_path_list
